Cargo.toml

```diff
@@ -311,6 +311,7 @@ ethexe-prometheus = { path = "ethexe/prometheus", default-features = false }
 ethexe-validator = { path = "ethexe/validator", default-features = false }
 ethexe-rpc = { path = "ethexe/rpc", default-features = false }
 ethexe-common = { path = "ethexe/common", default-features = false }
+ethexe-compute = { path = "ethexe/compute" }
```

Comment:
default features false (despite the fact that it may not have any atm)

Reply:
done